js_modules/dagster-ui/packages/ui-components/src/fonts/GlobalGeistMono.tsx

```diff
@@ -16,7 +16,6 @@ export const GlobalGeistMono = createGlobalStyle`
     font-family: 'Geist Mono';
     font-weight: 100 900;
     font-style: normal;
-    font-variant-ligatures: none;
```

Comment:
@salazarm It seems like this isn't working, but I may be missing something. Did it work in some cases and not in others?